packages/cactus-plugin-satp-hermes/src/main/typescript/core/stage-services/server/stage0-server-service.ts

```diff
@@ -1,30 +1,3 @@
-import {
-  PreTransferCommenceRequestMessage,
-  PreTransferCommenceResponseMessage,
-  PreTransferVerificationAndContextEstablishmentRequest,
-  PreTransferVerificationAndContextEstablishmentResponse,
-} from "../../../generated/proto/cacti/satp/v02/stage_0_pb";
-import {
-  MessageType,
-  CommonSatp,
-} from "../../../generated/proto/cacti/satp/v02/common/message_pb";
-import { SessionData } from "../../../generated/proto/cacti/satp/v02/common/session_pb";
-import { SATP_VERSION } from "../../constants";
-import {
-  bufArray2HexStr,
-  getHash,
-  sign,
-  verifySignature,
-} from "../../../gateway-utils";
-import { TransferClaims } from "../../../generated/proto/cacti/satp/v02/common/message_pb";
-import {
-  TimestampType,
-  getMessageHash,
-  getMessageTimestamp,
-  saveHash,
-  saveSignature,
-} from "../../session-utils";
-import { SATPSession } from "../../../core/satp-session";
 import {
```

Comment:
im not sure why you are deleting stage 0 service? if functionality is not being used (which is likely the case), please have placeholders

Reply:
the code deleted was not being used